test/router-options.test.js

```diff
@@ -104,9 +104,12 @@ test('Should expose router options via getters on request and reply', t => {
   }, (req, reply) => {
     t.equal(reply.context.config.url, '/test/:id')
     t.equal(reply.context.config.method, 'GET')
-    t.equal(req.routeConfig.url, '/test/:id')
-    t.equal(req.routeConfig.method, 'GET')
-    t.same(req.routeSchema, expectedSchema)
+    t.equal(req.routeConfig.url, '/test/:id') // TODO: deprecate on @v5
+    t.equal(req.routeConfig.method, 'GET') // TODO: deprecate on @v5
+    t.same(req.routeSchema, expectedSchema) // TODO: deprecate on @v5
```

Comment:
We should move these assertions to a test case that does the same check as https://github.com/fastify/fastify/blob/main/test/listen.deprecated.test.js

Reply:
created a file ptal